README.md

```diff
@@ -8,6 +8,7 @@ Python3 event-driven modular IRC bot!
 * [scrypt](https://pypi.python.org/pypi/scrypt)
 * [suds](https://pypi.python.org/pypi/suds-jurko)
 * [twitter](https://pypi.python.org/pypi/twitter)
+* [pyopenssl](https://pypi.org/project/pyOpenSSL/)
```

Comment:
It's the SSL library required if your python doesn't come installed with it, a fresh install of Python 3.7 on CentOS left me without the ability to use ssl until i installed pyopenssl.

Reply:
In which way was SSL failing?

Reply:
There was no library for python to use, it simply wouldn't connect via SSL. I don't know if it's because I built it from source, but there's no harm in having itthere as an edge case, worst that happens is that it's already installed.